flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NetworkEnvironment.java

```diff
@@ -98,15 +99,15 @@ private NetworkEnvironment(
 		ConnectionManager connectionManager,
 		ResultPartitionManager resultPartitionManager,
 		TaskEventPublisher taskEventPublisher,
-		IOManager ioManager) {
+		ResultPartitionFactory resultPartitionFactory) {
 
 		this.config = config;
 		this.networkBufferPool = networkBufferPool;
 		this.connectionManager = connectionManager;
 		this.resultPartitionManager = resultPartitionManager;
 		this.taskEventPublisher = taskEventPublisher;
-		this.ioManager = ioManager;
 		this.isShutdown = false;
+		this.resultPartitionFactory = resultPartitionFactory;
```

Comment:
`this.resultPartitionFactory = resultPartitionFactory` in front of `this.isShutdown = false`?